### PR TITLE
fix: bring up services with deps with --no-deps

### PR DESCRIPTION
**Pull request by nicksieger**
**What I did**

Don't fail on not finding dependent services because they were put in the disabled slice.

**Related issue**
#9427.

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**
Photo by Artem Lysenko from Pexels

### CODE REVIEW

**Comment by maxcleme** [comment]
LGTM

File: pkg/e2e/start_stop_test.go [hunk 1]
```diff
@@ -131,6 +131,12 @@ func TestStartStopWithDependencies(t *testing.T) {
 		assert.Assert(t, strings.Contains(res.Combined(), "e2e-start-stop-with-dependencies-foo-1"), res.Combined())
 	})
 
+	t.Run("Up no-deps links", func(t *testing.T) {
+		_ = c.RunDockerComposeCmd("--project-name", projectName, "down")
+		res := c.RunDockerComposeCmd("-f", "./fixtures/links/compose.yaml", "--project-name", projectName, "up", "--no-deps", "-d", "foo")
+		assert.Assert(t, strings.Contains(res.Combined(), "Container e2e-start-stop-with-dependencies-foo-1  Started"), res.Combined())
+	})
```
Comment (glours) [inline pkg/e2e/start_stop_test.go:138]: Does it make sens to add a `compose ps` command and check that there isn't any `links-bar-1` container running, just to be sure we don't start unexpected sevices?

**Review by glours — APPROVED** [review]
LTGM
It seems that you didn't  sign-off your commits (legally needed to be able to merge your contribution), click on the `DCO` check if you need instructions

File: pkg/e2e/start_stop_test.go (hunk 1 above)
Comment (nicksieger) [reply pkg/e2e/start_stop_test.go:138]: Sounds like a good additional check to put in the test!

**Comment by glours** [comment]
Ahahaha my bad I enabled auto-merge 🤦‍♂️ 
Let's add the test in a follow up PR

**Pull request merged**